### ISSUE TITLE
Configure Indaba 2020 event and application Form

### ISSUE THREAD

**Issue by avishkar58** [deep-learning-indaba/Baobab#606]
Depends on #601 

Create a migration that inserts the following new records into our database:
- Add an [event](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/events/models.py#L12) with the following data:
  - name: Deep Learning Indaba 2020
  - description: The Deep Learning Indaba 2020, Tunis, Tunisia
  - start_date: 23 August 2020
  - end_date: 28 August 2020
  - key: indaba2020
  - organisation_id: 1
  - email_from: baobab@deeplearningindaba.com
  - url: http://www.deeplearningindaba.com
  - application_open: 1 March 2020
  - application_close: 17 April 2020
  - review_open: 25 April 2020
  - review_close: 15 May 2020
  - selection_open: 15 May 2020
  - selection_close: 1 June 2020
  - offer_open: 1 June 2020
  - offer_close: 31 July 2020
  - registration_open: 1 June 2020
  - registration_close: 31 July 2020
  - event_type: EVENT

- Add an [application_form](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L4) with the following data
  - event_id: The one created above!
  - nominations: False

- Configure the [Sections](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L20) and [Questions](https://github.com/deep-learning-indaba/Baobab/blob/ed9ef8986eeb5d725ce24cffdbbec15b7224c83a/api/app/applicationModel/models.py#L35) according to [this document](https://docs.google.com/document/d/1KMiZK6kYAImroO07865Elei7jQ073j_dnbbiaGoJUzo/edit?usp=sharing), noting the following conventions:
  - The `options` field for `multi-choice` and `multi-checkbox` type questions are specified using a JSON array of objects, which looks something like the example below. The document lists the `label` for each entry, and the convention is that the `value` is the same as the `value` but in lowercase, with a dash (-) replacing spaces and words like "yes", "and", "or", "only" and punctuation are removed.

```
[
  {"value": "transport-accommodation", "label": "Yes, transport and accommodation"}, 
  {"value": "transport", "label": "Yes, transport only"}, 
  {"value": "accommodation", "label": "Yes, accommodation only"}, 
  {"value": "none", "label": "None"}
]
```

Have a look at [previous migrations](https://github.com/deep-learning-indaba/Baobab/blob/develop/api/migrations/versions/87213d612eaf_eeml_application_form.py) for inspiration (although note that the data model may have changed since then!)

**Comment by zakkymustafa** [comment]
@avishkar58 I'm configuring the Sections and Questions, I can see a dependency column from each...like which dtype is it, an elaboration on what it does maybe

**Comment by avishkar58** [comment]
Hi @zakkymustafa - thanks for jumping in on this issue :) 
The `depends_on_question_id` is an integer foreign key that links to the id of the question table - it says that "this section/question depends on this other question". The `show_for_values` field has type JSON and we expect it to be an array of strings. Have a look at [this migration](https://github.com/deep-learning-indaba/Baobab/blob/develop/api/migrations/versions/14d53fa0c9c8_.py) from @OdendaalG  that was just merged into develop which sets up an application form using the dependency fields. 
Hope this answers your question - let me know if you need any more information?

**Comment by avishkar58** [comment]
Hi @zakkymustafa - are you still working on this issue? We're hoping to have it done asap so that we can start testing, please unassign yourself if you aren't able to complete it in a day or 2...

**Comment by zakkymustafa** [comment]
Thanks for your patience, finishing it up

**Comment by avishkar58** [comment]
Great thanks! :) Let me know if you need any more information :)
